woocommerce/woocommerce-ios
# Remove Products M1, M2, and readonly variants feature flags

Fixes #2703 

## Changes

- Removed 3 feature flags that have been fully launched for a few releases: `editProducts`, `editProductsRelease2`, and `readonlyProductVariants`
- Updated the related code to remove all M2 logic
- Updated unit tests, removed ones with M2 feature flag as `false`

## Testing

No user-facing changes are expected and the actions should be covered in unit tests, please just sanity check on the product details:

- Go to the Products tab
- Tap on an editable product
- Edit the product images
- Tap "Update" --> the product should be saved remotely

Update release notes:
